pkgs/applications/science/biology/bayescan/default.nix

```diff
@@ -34,6 +34,7 @@ stdenv.mkDerivation rec {
     homepage = "http://cmpg.unibe.ch/software/BayeScan";
     license = licenses.gpl3;
     maintainers = [ maintainers.bzizou ];
+    mainProgram = "bayescan_2.1";
```

Comment:
This may break when the package is updated to a new version

Reply:
Good catch! Fixed.